### PR TITLE
Add max-days and max-backups default value 0 means for PD

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9330
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: pd-configuration-file.md [hunk 1]
```diff
@@ -152,11 +152,13 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
 ### `max-days`
 
 + 日志保留的最长天数。
++ 如果未设置本参数或把此参数设置为默认值 `0`，PD 不清理日志文件。
```
Comment (TomShawn) [inline pd-configuration-file.md:155]:
```suggestion
+ 如果未设置本参数或把本参数设置为默认值 `0`，PD 不清理日志文件。
```

File: pd-configuration-file.md [hunk 2]
```diff
@@ -152,11 +152,13 @@ PD 配置文件比命令行参数支持更多的选项。你可以在 [conf/conf
...
 + 默认：0
 
 ### `max-backups`
 
 + 日志文件保留的最大个数。
++ 如果未设置本参数或把此参数设置为默认值 `0`，PD 会保留所有的日志文件。
```
Comment (TomShawn) [inline pd-configuration-file.md:161]:
```suggestion
+ 如果未设置本参数或把本参数设置为默认值 `0`，PD 会保留所有的日志文件。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by Oreoxmt** [comment]
@rleungx PTAL

**Pull request merged**